### ISSUE TITLE
Using Memberof with conditionals generates 500 error

### ISSUE THREAD

**Issue by wandering213** [modxcms/revolution#13167]
### Summary
When I try to use conditionals to output if a user is a member of a group I get a 500 error. If I don't include 'then/else the error is gone.

### Step to reproduce
```
[[!+modx.user.id:mo=`GroupName`:eq=`1`:then=`True`:else=`False`]]
```

### Observed behavior
Generates a 500 server error

### Expected behavior
I would expect it to output True or False

### Environment
MODX version 2.5.1, nginx 1.11.5, mysql 5.7.16, Ubuntu 16.04

**Comment by pixelchutes** [comment]
What happens if you remove the ```:eq=`1````?

**Comment by wandering213** [comment]
That resolves the error.

**Comment by OptimusCrime** [comment]
Without the eq statement, is the behavior still correct?

**Comment by wandering213** [comment]
Yes

**Comment by Jako** [comment]
A conditional could not combined directly with the next conditional as far as I know. There has to be a some conjunction between like `and:` or `or:`. But the Internal Server Error is somehow a bug and should be removed.

**Comment by OptimusCrime** [comment]
My thoughts exactly.

**Comment by OptimusCrime** [comment]
Traced the bug down to lines like these: https://github.com/modxcms/revolution/blob/2.x/core/model/modx/filters/modoutputfilter.class.php#L145

Conditional arrays should be `[1, '||', 1]`, `[0, '&&', 1]`, and similar. Problem happens when they do not follow the format `[value, operand, value, operand, value]` and so on. In this case the array is `[1, 1]` (two true after each other), which fails the eval function. 

I am going to submit a PR once I get a chance, but I think the approach would be to loop the array and make sure we have a int first and last, and alternating operands and ints between those.

**Comment by Mark-H** [comment]
Fixed in 2.6
